xbmc/guilib/TextureBundle.cpp

```diff
@@ -18,15 +18,10 @@
  *
  */
 
-#include "system.h"
 #include "TextureBundle.h"
 
-CTextureBundle::CTextureBundle(void)
-{
-  m_useXBT = false;
-}
-
-CTextureBundle::~CTextureBundle(void)
+CTextureBundle::CTextureBundle()
+  : m_useXBT{false}
```

Comment:
That is correct since c++11, also preferred since it breaks the build if you accidentally perform implicit conversion during init